src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/integration/IntegrationTestBase.kt

```diff
@@ -43,6 +51,25 @@ abstract class IntegrationTestBase {
   @Autowired
   lateinit var restrictedPatientRepository: RestrictedPatientsRepository
 
+  @Autowired
+  lateinit var hmppsQueueService: HmppsQueueService
+
+  @SpyBean
+  lateinit var domainEventPublisher: DomainEventPublisher
+
+  @SpyBean
+  lateinit var telemetryClient: TelemetryClient
+
+  @SpyBean
+  lateinit var prisonApiApplicationGateway: PrisonApiApplicationGateway
+
+  val offenderEventQueue by lazy {
+    hmppsQueueService.findByQueueId("offenderevents") ?: throw (RuntimeException("Queue offenderevents not found"))
+  }
+  val domainEventQueue by lazy {
+    hmppsQueueService.findByQueueId("domainevents") ?: throw (RuntimeException("Queue domainevents not found"))
+  }
```

Comment:
Consolidated SpyBeans into the base test class to prevent Spring from needing to restart (should speed up tests).